info.xml

```diff
@@ -8,28 +8,30 @@
     <author>SYSTOPIA</author>
     <email>info@systopia.de</email>
   </maintainer>
-  <releaseDate></releaseDate>
+  <releaseDate/>
   <version>2.3-dev</version>
   <develStage>dev</develStage>
   <compatibility>
-    <ver>5.2</ver>
```

Comment:
if you ask me, I would set compatibility to `5.69` (is the first release with smarty3, the main reason for this PR), but i don't know how you handle compatibility for your extensions.

As what concerns us, this fix is critical.. in case you have to delay the launch of this, please let us know we will need to fork the ext and make a release (temporary) to deploy in our prod sites. tx!

Reply:
I moved the fix to the `2.3` milestone and a version `2.3.0` could be released soon (dropping the old `5.2` core compatibility). There's one more PR pending a review for `2.3` as of now - would you be able to have a look at #145, @sluc23? All other issues could then be postponed to the `2.4` milestone.

Reply:
we don't use LineItems nor PriceSets, so I won't be the best to test #145 sorry.. but that PR has already 2 years old.. maybe @jaapjansma can confirm if is still an issue